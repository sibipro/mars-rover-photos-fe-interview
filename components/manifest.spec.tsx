import Manifest from './manifest';
import renderer from 'react-test-renderer'

test('manifest component is rendered', () => {
    const component = renderer.create(<Manifest display={true} roverName="Mock Rover" />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot()
})