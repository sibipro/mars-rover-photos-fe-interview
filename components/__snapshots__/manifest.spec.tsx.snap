// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`manifest component is rendered 1`] = `
<div>
  <h3>
    Manifest
  </h3>
</div>
`;
